console/src/hardware/device/external.ts

```diff
@@ -7,5 +7,9 @@
 // License, use of this software will be governed by the Apache License, Version 2.0,
 // included in the file licenses/APL.txt.
 
+import { notificationAdapter } from "@/version/Updater";
```

Comment:
You should be import `Version` from "@/version" instead of importing one file in the package.